tests/gnrc_ipv6_ext_opt/tests/01-run.py

```diff
@@ -0,0 +1,619 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018 Freie Universität Berlin
```

Comment:
```suggestion
# Copyright (C) 2020 Freie Universität Berlin
```

Reply:
I'll update the year range. Most of the helper functions were copied from `tests/gnrc_ipv6_ext`.